internal/ir/ir.go

```diff
@@ -397,6 +426,32 @@ func (*Struct) member()          {}
 func (s *Struct) Name() string   { return s.name }
 func (s *Struct) String() string { return fmt.Sprintf("type %s struct", s.Name()) }
 
+func (*Object) value()         {}
+func (o *Object) Name() string { return "" }
+func (o *Object) String() string {
+	values := make([]string, 0)
+	for _, value := range o.Values {
+		values = append(values, value.Name())
+	}
+
+	buf := bytes.NewBufferString(o.Comment)
+	if o.Type != nil {
+		buf.WriteString(fmt.Sprintf("(%s)", o.Type.Name()))
+	}
+
+	if len(o.Values) > 0 {
+		buf.WriteString(fmt.Sprintf("[%s]", strings.Join(values, ",")))
+	}
+
+	return buf.String()
+}
```

Comment:
This `String` method seems to complex to leave in this file, I think that should be moved to `ir/print.go` file with other complex `String` implementations.